homework/react_2025/react-1-hw-1/components/destination/AddWishlistItem.js

```diff
@@ -2,36 +2,40 @@
 
 import { useState } from 'react';
 import styles from './destination.module.css';
-export const AddWishlistItem = ({
-  onAddWishlistItem,
-}) => {
-  const [thumbnail, onThumbnailChange] = useState('/destination/image-europa.png');
-  // TASK - React 1 week 3
-  // 1. Add a useState for the handling the <input id="customWishlist" type="text" />
-  // 2. Connect the onThumbnailChange to the <select>
+
+export const AddWishlistItem = ({onAddWishlistItem}) => {
+  const [thumbnail, setThumbnail] = useState('');
+  const [inputValue, setInputValue] = useState('');
 
   const onAddItemPressed = () => {
-    // TASK - React 1 week 3
-    // implement this function
-    // Clear the <input/> field on button press
-    // pass the thumbnail and the name from the input to the onAddWishlistItem function
-    // call the onAddWishlistItem here
+    if (!inputValue.trim()) return;
+    onAddWishlistItem(inputValue, thumbnail);
+    setInputValue('');
   }
 
 
   return (
     <div className={styles.addWishlistItem}>
       <p>Add custom planet to wishlist</p>
       <label htmlFor="customWishlist">Wishlist item name</label>
-      <input id="customWishlist" type="text" />
+      <input
+        id="customWishlist"
+        type="text"
+        value={inputValue}
+        onChange={e => setInputValue(e.target.value)}
+      />
       <label htmlFor="customWishlistThumbnail">Wishlist item thumbnail</label>
-      <select id="customWishlistThumbnail" >
-        <option value="/destination/image-europa.png">EUROPA</option>
```

Comment:
the logic here is close but not 100% there. 
planets selected/unseleceted with the round buttons should be added/removed from the wishlist